### PR TITLE
Goldman Sachs Intern Hiring 2020

### PR DESCRIPTION
**Pull request by anushreekhanna**
This is a question that was a part of my intern hiring process that took place in August.

### CODE REVIEW

**Review by samagragupta — COMMENTED** [review]
Give an example of input and output case.

**Comment by anushreekhanna** [comment]
Done.

File: Company-Preperation/Goldman-Sachs-2020/GoldmanSachs1.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'''
```
Comment (Gurubalan-GIT) [inline Company-Preperation/Goldman-Sachs-2020/GoldmanSachs1.py:1]: Great work! Do keep pushing more for company hiring processes you've gone through! 💯

**Comment by Gurubalan-GIT** [comment]
> Give an example of input and output case.

@samagragupta Thanks for the reviews out here, been seeing your contributions a lot of times and 
I appreciate what you doing here!

**Pull request merged**